operator/option/config.go

```diff
@@ -503,6 +511,7 @@ func (c *OperatorConfig) Populate() {
 	c.CiliumPodLabels = viper.GetString(CiliumPodLabels)
 	c.RemoveCiliumNodeTaints = viper.GetBool(RemoveCiliumNodeTaints)
 	c.SetCiliumIsUpCondition = viper.GetBool(SetCiliumIsUpCondition)
+	c.AllowedAnnotations = viper.GetStringSlice(AllowedAnnotations)
```

Comment:
Oh this is not sufficient. We also need to add the flag [here](https://github.com/cilium/cilium/blob/9366a140f16fdfde6af44cd14686f3a5a73cb402/operator/flags.go#L335-L338)

Reply:
Sorry for that, added the flag